temporal_capi/src/plain_year_month.rs

```diff
@@ -142,6 +142,28 @@ pub mod ffi {
         pub fn equals(&self, other: &Self) -> bool {
             self.0 == other.0
         }
+        pub fn compare(one: &Self, two: &Self) -> i32 {
+            Self::compare_iso_year_month(
+                one.iso_year(),
+                one.iso_month(),
+                two.iso_year(),
+                two.iso_month(),
+            )
+        }
+
+        pub fn compare_iso_year_month(year1: i32, month1: u8, year2: i32, month2: u8) -> i32 {
```

Comment:
suggestion: write this as a match on `(year1, month1).cmp((year2, month2))` instead

And link to the spec entry

Reply:
also, thought: I don't think it would be a bad idea to just have a method return std::cmp::Ordering here, that converts to the integer anyway, and makes it easier to add comparator support later